src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -327,6 +328,62 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
             return Text(runner.GetFormattingInfo(), irContext, args, runner.CancellationToken);
         }
 
+        public static FormulaValue TextToBlob(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            runner.CancellationToken.ThrowIfCancellationRequested();
+
+            if (args[0].IsBlank())
+            {
+                return new BlobValue(irContext);
+            }

```

Comment:
Since you use ReturnBehavior.ReturnBlankIfAnyArgIsBlank in the wrapper in LibraryUnary.cs, this should never be hit. And we should not return a BlobValue in this case - conversion functions propagate blank values (i.e., blank in, blank out)